src/wallet/rpcdump.cpp

```diff
@@ -768,7 +768,8 @@ UniValue importelectrumwallet(const JSONRPCRequest& request)
     if (!reserver.reserve()) {
         throw JSONRPCError(RPC_WALLET_ERROR, "Wallet is currently rescanning. Abort existing rescan or wait.");
     }
-    pwallet->ScanForWalletTransactions(chainActive[nStartHeight], nullptr, reserver, true);
+    const CBlockIndex *stop_block, *failed_block;
+    pwallet->ScanForWalletTransactions(chainActive[nStartHeight], nullptr, reserver, failed_block, stop_block, true);
```

Comment:
I don't think these changes belong in 13076

Reply:
It's fine, `importelectrumwallet` is a Dash-specific rpc.